2주차/수 찾기.py

```diff
@@ -0,0 +1,13 @@
+import sys
+
+n = int(sys.stdin.readline().strip())
+numbers = set(map(int, input().split()))  
+
+m = int(sys.stdin.readline().strip())
+question = list(map(int, input().split()))
+
+for i in question:
+    if i in numbers:
+        print(1)
+    else:
+        print(0)
```

Comment:
```suggestion
    print(int(i in numbers))
```
불리언을 int로 바꾸면 True는 1, False는 0이니까 이렇게도 쓸 수 있어요
굳이 이렇게 쓸 필요는 없고, 백준같은데서 다른 사람 코드 보면 이해할수 있도록 알아만 두세요